### PR TITLE
fix(desktop): add Android connection guidance

### LINKED ISSUE #206: [Feature] 在 Android 自动化中提供 ADB 连接和 USB 调试引导
## 使用场景

用户在「设置 - 自动操作 - Android 自动化」中启用能力，希望让 agent 操作已通过 USB 连接的 Android 手机。

## 当前问题

Cindy 可以自动准备 platform-tools，并会检测设备的未连接、未授权和 offline 状态，但页面只显示 ADB 技术状态，没有说明用户下一步需要做什么。

对未使用过 ADB 的用户，「ADB 已就绪，但未连接已授权的 Android 设备」和「USB debugging 尚未授权」无法转化为可执行操作，容易误以为 Cindy 配置失败。

## 期望结果

在 Android 自动化设置卡中提供面向普通用户的连接引导，不要求用户安装 Android SDK 或在终端执行 ADB 命令。

- 当没有可用设备时，提供「连接 Android 设备」帮助入口，说明：
  1. 使用支持数据传输的 USB 数据线连接手机和电脑；
  2. 在手机「设置 - 关于手机」连续点击版本号以开启开发者选项；
  3. 在「设置 - 开发者选项」开启「USB 调试」；
  4. 保持手机解锁，回到 Cindy 点击「重新检查」。
- 当设备为 `unauthorized` 时，明确提示在手机弹窗中选择「允许 USB 调试」，并建议勾选「始终允许使用这台电脑」。
- 当设备为 `offline` 时，提供简短排查：重新插拔数据线、保持设备解锁后重新检查。
- 引导应说明 Cindy 会自动准备 ADB；「自定义 ADB 路径」仅作为高级选项，避免用户误以为必须自行安装或配置 ADB。
- 引导内容随状态展示，不遮挡已连接设备的选择和 ADB 路径配置。
- 补齐中英文及现有受支持语言的对应文案，并为状态到引导内容的映射添加单元测试。

## 备选方案

仅在帮助文档增加 ADB 教程。该方案发现性较弱，无法在用户实际遇到「未连接 / 未授权」状态时提供对应操作，因此不作为首选。

## 实现建议

1. 在 Desktop Renderer 的 Android 自动化设置卡中，根据 `NO_DEVICE`、`DEVICE_UNAUTHORIZED` 和 `DEVICE_OFFLINE` 渲染对应引导入口与内容；保持现有设备选择、状态刷新和 ADB 路径配置的行为不变。
   - 验证：为各状态的引导呈现补充 Renderer 单元测试，并人工检查 Light / Dark 模式下的布局与可读性。
2. 为引导内容补齐 zh-CN、en、ja、ko 文案，并确认 i18n key 在各语言文件中一致。
   - 验证：运行对应的 i18n 校验与 Desktop 定向测试。

### PR DESCRIPTION
**Pull request by xSharkM**
### 摘要

为 Android 自动化设置补充按设备状态变化的连接指引，解决首次使用者只看到 `NO_DEVICE`、`DEVICE_UNAUTHORIZED` 或 `DEVICE_OFFLINE` 状态，却不知道下一步如何连接手机的问题。

### 变更类型

- [ ] `feat` 新功能
- [x] `fix` 缺陷修复
- [ ] `refactor` / `perf` 重构或性能优化
- [ ] `docs` / `test` / `chore` 文档、测试或工程维护
- [ ] 其他：

### 范围

- 关联 Issue / 需求：Closes #206
- 本 PR 包含：
  - 无设备时展示 USB、开发者选项、USB 调试和重新检查步骤
  - 未授权与离线设备分别展示可操作的恢复提示
  - 有可用设备时不展示误导性的重新连接指引
  - 明确 Cindy 会自动准备 ADB，自定义路径仅为高级设置
  - 补齐 zh-CN / en / ja / ko 文案与状态映射测试
- 明确不包含：不修改 ADB 安装、探测、设备连接或 Android runtime 逻辑
- 用户可见变化：设置 → 电脑使用 → Android 自动化会根据当前设备状态显示连接指引
- 是否存在 breaking change：无

### UI 变化

- Android 自动化设置卡片增加使用现有主题 token 的状态指引区域；请求方已在桌面端验证 Light / Dark 模式。
- 引用的设计规范：`docs/design-rules/DESIGN.md` §2 Layer System（沿用设置卡片层级与边框）、§3 Typography（12px 正文与标题层级）、§9 Light / Dark Dual-Mode Delivery Gate、§10 Theme System & Token Reference（所有颜色使用现有语义 token，无新增硬编码颜色）。

### 自动验证

```text
pnpm test:unit
结果：通过（全部 workspace）

pnpm --filter desktop run --if-present typecheck
结果：通过

pnpm check:i18n
结果：通过（仅既有非阻塞警告）

pnpm check:i18n-glossary
结果：通过

pnpm check:dco
结果：通过，1 个 commit 已签署

git diff --check
结果：通过
```

### 手工验证

- 平台：Desktop
- 路径：设置 → 电脑使用 → Android 自动化
- 已由请求方验证连接指引及 Light / Dark 显示正常。

### 未执行的验证

无。

### 风险分类

- [x] 无已知风险
- [ ] SQLite / migration
- [ ] system prompt
- [ ] 协议兼容
- [ ] 权限 / 安全 / 用户数据
- [ ] 原生层 / fingerprint / OTA
- [ ] 跨平台差异
- [ ] 其他：

### 影响与回滚

- 影响范围：仅 Desktop Renderer 的 Android 状态展示与 i18n 文案。
- 回滚 / 降级方式：回滚本提交即可恢复原有状态展示；不涉及数据迁移或协议变更。

### 提交前检查

- [x] 已 review 完整 diff
- [x] 每个 commit 都带 DCO 签名（`git commit -s`，见 [DCO](../DCO)）
- [x] UI 改动已在「UI 变化」注明引用的设计规范章节（不涉及 UI 则跳过）
- [x] 未提交凭证、令牌或授权文件
- [x] 已补充必要测试与多语言文案
- [x] 已确认测试结果或说明未执行原因

### CODE REVIEW

File: apps/desktop/src/renderer/components/settings/androidStatusPresentation.ts [hunk 1]
```diff
@@ -87,6 +89,28 @@ export function describeAndroidStatus(status: AndroidStatusSummary, t: TFunction
...
+): AndroidConnectionGuideKind | null {
+  if (!status?.adb_available) return null;
+
+  const hasReadyDevice = status.devices.some((device) => device.state === 'device');
+  if (hasReadyDevice) return null;
+
+  if (status.issue === 'DEVICE_UNAUTHORIZED') return 'unauthorized';
+  if (status.issue === 'DEVICE_OFFLINE') return 'offline';
+  if (status.issue === 'NO_DEVICE' || !status.issue) return 'connect';
+  return null;
```
[the remark this answers is not in the document]
Comment (xSharkM) [reply apps/desktop/src/renderer/components/settings/androidStatusPresentation.ts:120]: 已在 2316c93ea 修复：先识别已配置默认设备本身的 unauthorized/offline 状态；没有 ready 设备时再从 devices 推导可操作状态，最后才回退到聚合 issue。这样 stale default 导致的 NO_DEVICE 不会覆盖真实恢复动作。

File: apps/desktop/src/renderer/components/settings/__tests__/androidStatusPresentation.test.ts [hunk 2]
```diff
@@ -115,4 +116,48 @@ describe('android status presentation', () => {
...
+        adb_available: true,
+        adb_path: '/sdk/platform-tools/adb',
+        version: 'Android Debug Bridge version 1.0.41',
+        devices: [],
+        default_device_serial: null,
+        issue,
+      }),
+    ).toBe(guide);
+  });
+
```
[the remark this answers is not in the document]
Comment (xSharkM) [reply apps/desktop/src/renderer/components/settings/__tests__/androidStatusPresentation.test.ts:136]: 已在 2316c93ea 补充回归测试：覆盖 stale configured default + NO_DEVICE + unauthorized/offline，以及默认设备异常但另一设备 ready 的场景。定向测试 15/15、完整 pnpm test:unit 和 Desktop typecheck 均通过。

**Comment by MagicLizi** [comment]
本 PR 改动命中 UI 路径（`ComputerUseSection.tsx` / `androidStatusPresentation.ts`）但 description 未附界面效果证据——建议补充改动后效果：截图/录屏，或改动后界面的 HTML 页面（```html 代码块、.html 附件或在线预览链接），便于确认界面符合 DESIGN.md 设计规范。

**Review by MagicLizi — APPROVED** [review]
Code review passed: well-structured fix with clean separation of presentation logic (pure testable function), complete i18n across 4 locales, all CSS via semantic tokens for dual-mode compatibility, good test coverage for edge cases. No security concerns. chatgpt-codex-connector timed out (128 min > 20 min cap) — its review may arrive post-merge.
